soc/arm/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -361,3 +361,21 @@ config NRF_ENABLE_ICACHE
 	bool "Enable the instruction cache (I-Cache)"
 	depends on SOC_NRF52832 || SOC_NRF52833 || SOC_NRF52840
 	default y
+
+config NRF52_ANOMALY_132_DELAY_US
+	int "Anomaly 132 workaround delay (microseconds)"
+	default 220
```

Comment:
Should we have a range here, to prevent high numbers that might have bad effects, (e.g. to watchdog)?

Reply:
done